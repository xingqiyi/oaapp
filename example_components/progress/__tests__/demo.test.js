import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('progress');
