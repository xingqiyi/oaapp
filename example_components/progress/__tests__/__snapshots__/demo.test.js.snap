// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/progress/demo/basic.md correctly 1`] = `
<div
  class="progress-container"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="30"
    class="am-progress-outer am-progress-fixed-outer"
    role="progressbar"
  >
    <div
      class="am-progress-bar"
      style="width:30%;height:0;"
    />
  </div>
  <div
    style="height:36px;"
  />
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="40"
    class="am-progress-outer am-progress-hide-outer"
    role="progressbar"
  >
    <div
      class="am-progress-bar"
      style="width:0;height:0;"
    />
  </div>
  <div
    class="show-info"
  >
    <div
      class="progress"
    >
      <div
        aria-valuemax="100"
        aria-valuemin="0"
        aria-valuenow="50"
        class="am-progress-outer"
        role="progressbar"
      >
        <div
          class="am-progress-bar"
          style="width:50%;height:0;"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
    >
      50%
    </div>
  </div>
  <a
    aria-disabled="false"
    class="am-button am-button-inline"
    role="button"
    style="margin-top:20px;"
  >
    <span>
      (+-)10
    </span>
  </a>
</div>
`;
